lib/board.rb

```diff
@@ -0,0 +1,11 @@
+class Board
+  attr_reader :grid
+
+  def initialize(grid)
+    @grid = grid
+  end
+
+  def make_move(mark, position)
+    @grid[position - 1] = mark
+  end
```

Comment:
Is `make_move` the right name? What I mean is - are boards making moves? Just nitpicking, but the "does it make sense for X to do Y" is a useful test when picking names of functions.

Reply:
That's a really good point and thanks for the tip on picking function names! I'll change this